test/fixtures/admin_settings.yml

```diff
@@ -0,0 +1,22 @@
+admin_setting_3:
```

Comment:
I left out `guest_downloading_off` and `disable_filtering` which will be removed by [AO3-5474](https://otwarchive.atlassian.net/browse/AO3-5474) and [AO3-5473](https://otwarchive.atlassian.net/browse/AO3-5473).